Modelica/ComplexBlocks.mo

```diff
@@ -1356,10 +1355,10 @@ An error occurs if the elements of the input <code>u</code> is zero.
     block PolarToComplex "Converts polar representation to complex"
       extends Modelica.ComplexBlocks.Interfaces.ComplexSO;
       Blocks.Interfaces.RealInput len annotation (Placement(transformation(
-              extent={{-130,40},{-90,80}}), iconTransformation(extent={{-140,40},
+              extent={{-140,40},{-100,80}}),iconTransformation(extent={{-140,40},
                 {-100,80}})));
       Blocks.Interfaces.RealInput phi(unit="rad") annotation (Placement(transformation(
-              extent={{-130,-80},{-90,-40}}), iconTransformation(extent={{-140,
+              extent={{-140,-80},{-100,-40}}),iconTransformation(extent={{-140,
```

Comment:
Same question: Why are these connectors moved?

Reply:
All these connectors are now moved on the diagram layer (only): 

![image](https://user-images.githubusercontent.com/4184218/72367768-124ec780-36fd-11ea-8262-56bfbc01820b.png)


They were not aligned correctly before:

![image](https://user-images.githubusercontent.com/4184218/72367645-d87dc100-36fc-11ea-9af0-933ecf2ca7bb.png)

Reply:
I consider this change (somehow) relevant to this PR, as it cleans up the diagram layer.